### ISSUE TITLE
How to create a Skype-only live account?

### ISSUE THREAD

**Issue by vlad-at-work** [ocilo/skype-http#96]
This repo is pretty active but the demo doesn't function without an old account; what gives?

I have one as I registered it ages ago, but after some research it looks like you MUST have a MS Live account when you register for Skype.

Is there some gotcha to this?

**Comment by mitchcapper** [comment]
It works with old non MSA accounts without an issue.  What error are you getting?

**Comment by vlad-at-work** [comment]
@mitchcapper That's what I mean. Is this repo only good for old non MSA accounts/usernames? As in, any Skype registered as live:something won't work as per the documentation?

**Comment by mitchcapper** [comment]
@vladdypwnz you are correct it does say MSA accounts will not work, however I just tested one and it  worked fine.
Please run the following commands then paste the error you get after it gives the login prompt:
```
git clone https://github.com/demurgos/skype-http
cd skype-http
npm install
npm start
```
